applications/OptimizationApplication/python_scripts/algorithms/algorithm_nesterov_accelarated_gradient.py

```diff
@@ -0,0 +1,52 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+from KratosMultiphysics.OptimizationApplication.utilities.logger_utilities import time_decorator
+from KratosMultiphysics.OptimizationApplication.algorithms.algorithm_steepest_descent import AlgorithmSteepestDescent
+
+
+def Factory(model: Kratos.Model, parameters: Kratos.Parameters, optimization_problem: OptimizationProblem):
+    return AlgorithmNesterovAcceleratedGradient(model, parameters, optimization_problem)
+
+class AlgorithmNesterovAcceleratedGradient(AlgorithmSteepestDescent):
+    """
+        A classical steepest descent algorithm to solve unconstrainted optimization problems.
+    """
```

Comment:
Documentation. I would add publication references as well.

Reply:
You mean class description or I should add it to oyur documentation?

Reply:
I have added class description with paper link